wiki/en/Running-a-Server.md

```diff
@@ -283,58 +246,61 @@ When the Server is running, the operating system will show an icon in the system
+It is highly recommended to test your Server on a **Public Directory** first so as to narrow down any subsequent problems in unregistered mode.
 
-# Backing up the Server
+### Setting up a Server behind a home router
 
-_Note that headless Servers do not use `.ini` files. All configuration is given as command line options._
+If you set up your server at home, you will probably need to change some settings in your router/firewall:
 
-{% include_relative Include-Backing-Up.md %}
+#### Port forwarding
 
-# Command line options
+People from outside your home network will not be able to see things inside it. To let external Jamulus clients connect to your server, you need to set up port forwarding in your Router's settings. The exact setup differs for every router. For help see your Router's documentation or [portforward.com](https://portforward.com).
 
-Most common functions in Jamulus can be set using the GUI, but these and others can also be set using options given in a terminal window. Exactly how you do this will depend on your operating system.
+**Note:** The default port for the current version of Jamulus is **UDP** (not TCP) port **22124**. You will usually forward the port **22124** from outside your network to the port **22124** of the machine running the Server.
 
-For example on Windows, to use a specific settings file, right-click on the Jamulus shortcut and choose "Properties" > Target. Add the necessary arguments to Jamulus.exe:
+**Note:** Your home router may change the IP address of the machine you're running your Server on. Depending on your router, you might need to give this machine a static IP address (often under DHCP settings of your router).
 
-```shell
- "C:\Program Files\Jamulus\Jamulus.exe" --serverbindip 192.168.0.100
-```
+#### Getting the external IP
 
-For macOS, start a Terminal window and run Jamulus with the desired options like this:
+To allow others to connect to your Server from the internet, get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip) and give it to them.
+You yourself should connect using the local network (LAN) address of the machine the Server is running on. If you are running a Client on the same machine as your Server, that would be `localhost` or `127.0.0.1`.
 
-```shell
- /Applications/Jamulus.app/Contents/MacOS/Jamulus --serverbindip 192.168.0.100
-```
 
-**Note**: Command line options will set the Server's defaults at startup. You can override them with their corresponding GUI controls while the Server is running.
+#### Dynamic DNS and why you will probably need it
 
-{% include_relative Include-Server-Commands.md %}
+Most domestic internet connections change their external IP address after a short period, you might want to set up a "dynamic DNS" which gives you a static (sub-)domain you can access and share with others. For further information, please look if your router already provides this feature or use an external DynamicDNS provider.
```

Comment:
Don't you always have to use an external provider for dynamic DNS (unless you roll your own if you have your own named)? This is sort of why I'd prefer just removing this section. But, otherwise:

```suggestion
Most domestic internet connections change their external IP address after a short period. To avoid problems with this, you might want to set up "dynamic DNS" to get a static (sub-)domain you can share with others. Your router may have a built-in client you can use, or you can set one up on a computer on your network.
```

Reply:
Some routers support it (FritzBox by AVM) out of the Box; the manufacturer provides the DynamicDNS service. I think this is the first place to check. 

> To avoid problems with this

I don’t think there are many problems if you don’t have Dynamic DNS. It’s just very inconvenient without.

Reply:
By "this" I meant "change their external IP address after a short period" - so the problem of suddenly nobody being able to connect to your server. I'd regard that as a problem myself, but if it's not, then why make a big deal about ddns? :-) 

But again, I think all this just involves us in an area we don't need to get mixed up in: what routers do or do not do, etc.

Reply:
Yeah. I think the confusion is an edge case for FritzBoxes (Routers which are very common in Germany). I'll slightly modify the suggestion by @gilgongo